test/core/vm/actor/builtin/types/miner/test_utils.hpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "primitives/types.hpp"
+#include "testutil/outcome.hpp"
+#include "vm/actor/actor.hpp"
+#include "vm/actor/builtin/types/miner/expiration.hpp"
+#include "vm/actor/builtin/types/miner/sector_info.hpp"
+
+namespace fc::vm::actor::builtin {
+  using primitives::ChainEpoch;
+  using primitives::DealWeight;
+  using primitives::SectorNumber;
+  using primitives::TokenAmount;
+  using types::miner::ExpirationQueue;
+  using types::miner::SectorOnChainInfo;
+
+  class Utils {
```

Comment:
this class should be removed, prefer namespace

Reply:
Namespace is not suitable in this case because we have duplicate symbol error during compilation.
The way to use namespace is to move function implementation to .cpp file. Or just leave as it is.
What do you prefer?

Reply:
As I understand, we leave it as is

Reply:
duplicate symbols are resolved with `inline` or `constexpr`

Reply:
thank you, fixed